samples/basic/blinky/README.rst

```diff
@@ -6,9 +6,9 @@ Blinky Application
 Overview
 ********
 
-The Blinky example shows how to configure GPIO pins as outputs which can also be
-used to drive LEDs on the hardware usually delivered as "User LEDs" on many of
-the supported boards in Zephyr.
+The Blinky example shows how to drive LED devices attached to GPIO
```

Comment:
Change ``drive LED devices`` to ``configure LED devices``. To me, the word ``drive`` has hardware implications (voltage, resistors, and such), whereas the previous word ``configure`` feels better here since it's a software configuration we're discussing, right?

Reply:
Okay

Reply:
Actually, in this PR I have moved the configuration part inside the LED GPIO driver. So we don't configure pins in this sample anymore.